scripts/tools/metricProgrammable.yml

```diff
@@ -0,0 +1,512 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Tool Experimental Extension for Programmable Metrics"
+version: "1.8"
```

Comment:
@joshuaranjan please change all versions to "1.9"

Reply:
done.